src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -313,8 +313,8 @@ struct jit_uni_interpolate_kernel_f32 : public jit_uni_interpolate_kernel, publi
         std::vector<size_t> local_store_pool_vec_idxs = { static_cast<size_t>(vmm_dst.getIdx()) };
         local_store_pool_vec_idxs.insert(local_store_pool_vec_idxs.begin(), store_pool_vec_idxs.begin(), store_pool_vec_idxs.end());
 
-        emitters[seed]->emit_code({static_cast<size_t>(vmm_dst.getIdx())},
-                                  {static_cast<size_t>(reg_dst.getIdx()), static_cast<size_t>(offset)},
+        emitters[seed]->emit_code({static_cast<size_t>(vmm_dst.getIdx()), static_cast<size_t>(offset)},
+                                  {static_cast<size_t>(reg_dst.getIdx())},
```

Comment:
offset in store emitter is offset of dst gpr register, and parse together with reg_dst, so we do not need change here.

Reply:
already fixed